ci/scripts/run-vulnerability-scan/scan.sh

```diff
@@ -49,9 +74,11 @@ done
 
 IMAGES=$(echo "$IMAGES" | xargs -n1 | sort -u)
 
-aws ecr get-login-password --region "$AWS_DEFAULT_REGION" | docker login --username AWS --password-stdin "$ECR_REGISTRY"
-# Log in to dockerhub to run `docker scout`
-echo "$DOCKERHUB_ACCESS_TOKEN" | docker login -u "$DOCKERHUB_USERNAME" --password-stdin
+if [ "$MANUAL_RUN" != "true" ]; then
+  aws ecr get-login-password --region "$AWS_DEFAULT_REGION" | docker login --username AWS --password-stdin "$ECR_REGISTRY"
+  # Log in to dockerhub to run `docker scout`
+  echo "$DOCKERHUB_ACCESS_TOKEN" | docker login -u "$DOCKERHUB_USERNAME" --password-stdin
```

Comment:
If people are logged into docker desktop you shouldn't need to do this, you can do something like (probably much earlier, where you test if commands are installed):

```
if ! docker login; then
  echo "Error: You need to log into dockerhub in docker desktop"
  exit 1
fi
```
```

Reply:
This is for the Concourse job to run `docker scout`. If run locally we should get an error message to log in to dockerhub.

Reply:
🤦 Of course, yeah, thanks